docs/source/conf.py

```diff
@@ -17,13 +18,13 @@
 # If your extensions are in another directory, add it here. If the directory
 # is relative to the documentation root, use os.path.abspath to make it
 # absolute, like shown here.
-#sys.path.append(os.path.abspath('some/directory'))
+# sys.path.append(os.path.abspath('some/directory'))
```

Comment:
I should probably just delete all these commented out code blocks

Reply:
Actually I'm not sure, some of it isn't really dead code, but rather unused options.  I removed all of it in the latest commit, but if these should be kept I can simply revert that commit